lib/improver/tests/nowcasting/lightning/test_NowcastLightning.py

```diff
@@ -144,74 +221,27 @@ class Test__modify_first_guess(IrisTest):
     """Test the _modify_first_guess method."""
 
     def setUp(self):
-        """Create cubes with a single zero prob(precip) point.
-        The cubes look like this:
-        precipitation_amount / (kg m^-2)
+        """Create test cubes and plugin instance.
+        The cube coordinates look like this:
         Dimension coordinates:
-            time: 1;
             projection_y_coordinate: 3;
             projection_x_coordinate: 3;
-        Auxiliary coordinates:
-            forecast_period (on time coord): 4.0 hours (simulates UM data)
         Scalar coordinates:
-            forecast_reference_time: 2015-11-23 03:00:00
-        Data:
-        self.cube:
-            Describes the nowcast fields to be calculated.
-            forecast_period (on time coord): 0.0 hours (simulates nowcast data)
-            All points contain float(1.) except the
-            zero point [0, 1, 1] which is float(0.)
+            time: 2015-11-23 07:00:00
+            forecast_reference_time: 2015-11-23 07:00:00
+            forecast_period: 0 seconds
+
         self.fg_cube:
-            All points contain float(1.)
-        self.ltng_cube:
-            forecast_period (on time coord): 0.0 hours (simulates nowcast data)
-            All points contain float(1.)
+            Has 4 hour forecast period, to test impact at lr2
         self.precip_cube:
-            With extra coordinate of length(3) "threshold" containing
-            points [0.5, 7., 35.] mm hr-1.
-            All points contain float(1.) except the
-            zero point [0, 0, 1, 1] which is float(0.)
-            and [1:, 0, ...] which are float(0.)
+            Has extra coordinate of length(3) "threshold" containing
+            points [0.5, 7., 35.] mm h-1.
         self.vii_cube:
-            With extra coordinate of length(3) "threshold" containing
-            points [0.5, 1., 2.] kg m^-2.
-            forecast_period (on time coord): 0.0 hours (simulates nowcast data)
-            Time and forecast_period dimensions "sqeezed" to be Scalar coords.
-            All points contain float(0.)
+            Has extra coordinate of length(3) "threshold" containing
+            points [0.5, 1., 2.] kg m-2.
```

Comment:
Can you add back to the doc-string a little bit of information about self.cube and self.ltng_cube? The former (self.cube) in particular is now rather harder to trace as it changes name from template_cube. I just want to make it easy for the next programmer to work out what is being done here.